public/delete_post.php

```diff
@@ -0,0 +1,21 @@
+<?php 
+
+	require('database.php');
+
+	session_start();
+
+	$authorId = $_SESSION['author_id'];
+
+	$threadId = $_POST['thread_id'];
+
+	$content = $_POST['content'];
+
+	$db = new Database();
+
+	$query = "DELETE FROM posts WHERE author_id=, thread_id=, content=,";
```

Comment:
The syntax for deleting by multiple values would be:

```SQL
DELETE FROM posts WHERE author_id = 5 AND thread_id = 7 AND content = "something"
```

Notice the `AND`s instead of commas.

**However**, in this case you don't want to delete by content. You generally want to do the minimum amount that you need.

I would probably just delete it by `id`:

```SQL
DELETE FROM posts WHERE id = 5
```

However, this doesn't ensure that the user has permission to delete it. You could do that really easy in SQL by including a WHERE clause for author_id:

```SQL
DELETE FROM posts WHERE id = 5 AND author_id = 7
```